staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2701,13 +2712,24 @@ func (i *IngressDescriber) describeIngressV1beta1(ing *networkingv1beta1.Ingress
 		if def == nil {
 			// Ingresses that don't specify a default backend inherit the
 			// default backend in the kube-system namespace.
-			def = &networkingv1beta1.IngressBackend{
+			defaultBackend := &networkingv1beta1.IngressBackend{
 				ServiceName: "default-http-backend",
 				ServicePort: intstr.IntOrString{Type: intstr.Int, IntVal: 80},
 			}
-			ns = metav1.NamespaceSystem
+			_, err := i.client.CoreV1().Endpoints(metav1.NamespaceSystem).Get(context.TODO(), defaultBackend.ServiceName, metav1.GetOptions{})
+			if err == nil {
+				_, err := i.client.CoreV1().Services(metav1.NamespaceSystem).Get(context.TODO(), defaultBackend.ServiceName, metav1.GetOptions{})
+				if err == nil {
+					def = defaultBackend
+					ns = metav1.NamespaceSystem
+				}
+			}
+		}
```

Comment:
Is it right that defaultBackend will be showed if the endpoint named "default-http-backend" and the service named "default-http-backend" are exist in the metav1.NamespaceSystem when "def(ing.Spec.Backend)" is nil?  But I think that users will be confused  in the following case:
1. there are many ingresses in a namespace;
2. the endpoint named "default-http-backend" and the service named "default-http-backend" are exist in the namespace; 
3. but the above endpoint and service is not prepared for the ingress that the user described(it is prepared for another ingress).

So I think that defaultBackend should  be showed as  "none" when ing.Spec.Backend is nil.

Reply:
Agress, fixed as showing <default> in case ing.Spec.Backend is nil.